mappings/net/minecraft/client/particle/ParticleHeart.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS none/bnd
+CLASS none/bnd net/minecraft/client/particle/ParticleHeart
```

Comment:
Is that the damage heart, or breeding heart? While `Heart` couldn't be more fitting, `Love`, `Breeding` or maybe just `Red/FullHeart` would make sure nobody mistakes it?